lib/calendar_builder.rb

```diff
@@ -0,0 +1,43 @@
+class CalendarBuilder < Struct.new(:view, :date, :callback)
+    HEADER = %w[Sunday Monday Tuesday Wednesday Thursday Friday Saturday]
```

Comment:
I believe the Date class in Ruby has a constant with the day of the week, would be nice to use here.